content/en/blog/_posts/2017-08-00-Kompose-Helps-Developers-Move-Docker.md

```diff
@@ -3,6 +3,8 @@ title: " Kompose Helps Developers Move Docker Compose Files to Kubernetes "
 date: 2017-08-10
 slug: kompose-helps-developers-move-docker
 url: /blog/2017/08/Kompose-Helps-Developers-Move-Docker
+author: >
+  Charlie Drage (Red Hat)
 ---
 _Editor's note: today's post is by Charlie Drage, Software Engineer at Red Hat giving an update about the Kubernetes project Kompose._  
```

Comment:
I'd drop this note. We now just credit the person and optionally their organization.